src/server/api/index.ts

```diff
@@ -68,7 +68,7 @@ async function apiBuilder (cfg: APIConfig): Promise<FastifyTypebox> {
   const store = new Store(db)
   const modCheck = new ModerationChecker(store)
   const hookSystem = new HookSystem(store)
-  const apsystem = new ActivityPubSystem(cfg.publicURL, store, modCheck, hookSystem)
+  const apsystem = new ActivityPubSystem(cfg.publicURL, store, modCheck, hookSystem, server)
```

Comment:
Mind changing this to pass the `server.log` instead? I don't think we need the entire thing.